### ISSUE TITLE
Convolution bias that precedes batch normalization should be eliminated.

### ISSUE THREAD

**Issue by hccho2** [calmisential/TensorFlow2.0_ResNet#7]
For example,

```
self.conv1 = tf.keras.layers.Conv2D(filters=filter_num,
                                    kernel_size=(3, 3),
                                    strides=stride,
                                    padding="same")

```
shoud be

```
self.conv1 = tf.keras.layers.Conv2D(filters=filter_num,
                                    kernel_size=(3, 3),
                                    strides=stride,
                                    padding="same",use_bias=False)
```
